packages/fast-components-react-msft/src/text-action/text-action.tsx

```diff
@@ -48,14 +48,13 @@ class TextAction extends Foundation<
      */
     public render(): JSX.Element {
         return (
-            <div className={this.generateClassNames()}>
+            <div className={this.generateClassNames()} {...this.unhandledProps()}>
```

Comment:
Unhandled props should be before className or your overrides will not work.

Reply:
Makes perfect sense - good to know. We should include this in our documentation if it is not already there.